### PR TITLE
Files Provisioned Billing V2 updates

### PR DESCRIPTION
**Pull request by seanmcc-msft**
This PR corrects a typo that was made in https://github.com/Azure/azure-rest-api-specs/pull/30695, and feedback from the Files team.  This swagger version have not yet been released, so these are not breaking changes.

### CODE REVIEW

**Review by jalauzon-msft — APPROVED** [review]

File: specification/storage/data-plane/Microsoft.FileStorage/stable/2025-01-05/file.json [hunk 1]
```diff
@@ -328,19 +328,25 @@
...
               "x-ms-share-included-burst-iops": {
                 "x-ms-client-name": "ShareIncludedBurstIops",
                 "type": "integer",
                 "format": "int64",
-                "description": "ShareIncludedBurstIops"
+                "description": "Returns the calculated burst IOPS of the share."
+              },
+              "x-ms-share-max-burst-credits-for-iops": {
+                "x-ms-client-name": "MaxBurstCreditsForIops",
+                "description": "Returned the calculated maximum burst credits. This is not the current burst credit level, but the maximum burst credits the share can have.",
```
Comment (Mayuri-Gadsing) [inline specification/storage/data-plane/Microsoft.FileStorage/stable/2025-01-05/file.json:347]: Returns the calculated maximum burst credits

File: specification/storage/data-plane/Microsoft.FileStorage/stable/2025-01-05/file.json [hunk 2]
```diff
@@ -559,13 +565,13 @@
               },
               "x-ms-share-next-allowed-provisioned-iops-downgrade-time": {
                 "x-ms-client-name": "NextAllowedProvisionedIopsDowngradeTime",
-                "description": "Return timestamp for provisioned IOPS following existing rules for provisioned storage GiB.",
+                "description": "Returns the current share next allowed provisioend iops downgrade time.",
```
Comment (Mayuri-Gadsing) [inline specification/storage/data-plane/Microsoft.FileStorage/stable/2025-01-05/file.json:568]: provisioned

File: specification/storage/data-plane/Microsoft.FileStorage/stable/2025-01-05/file.json (hunk 1 above)
Comment (seanmcc-msft) [reply specification/storage/data-plane/Microsoft.FileStorage/stable/2025-01-05/file.json:347]: Fixed.

File: specification/storage/data-plane/Microsoft.FileStorage/stable/2025-01-05/file.json (hunk 2 above)
Comment (seanmcc-msft) [reply specification/storage/data-plane/Microsoft.FileStorage/stable/2025-01-05/file.json:568]: fixed.

File: specification/storage/data-plane/Microsoft.FileStorage/stable/2025-01-05/file.json [hunk 3]
```diff
@@ -630,12 +636,12 @@
                 "format": "date-time-rfc1123",
                 "description": "A UTC date/time value generated by the service that indicates the time at which the response was initiated."
               },
-              "x-ms-share-usage-bytes": {
+              "x-ms-file-share-usage-bytes": {
                 "type": "integer",
                 "format": "int64",
                 "description": "The \"live share\" portion of the data that the customer will be billed for in the soft-deleted capacity (logical storage size)."
```
Comment (Mayuri-Gadsing) [inline specification/storage/data-plane/Microsoft.FileStorage/stable/2025-01-05/file.json:642]: Description can be changed to - "Returned only for provisioned v2 file shares. Returns an approximate used storage size of the share, in bytes."

File: specification/storage/data-plane/Microsoft.FileStorage/stable/2025-01-05/file.json [hunk 4]
```diff
@@ -630,12 +636,12 @@
...
               },
-              "x-ms-share-snapshot-usage-bytes": {
+              "x-ms-file-share-snapshot-usage-bytes": {
                 "type": "integer",
                 "format": "int64",
                 "description": "The snapshot share portion of the data that the customer will be billed for in the soft-deleted capacity (this is the delta, or \"physical storage size\")."
```
Comment (Mayuri-Gadsing) [inline specification/storage/data-plane/Microsoft.FileStorage/stable/2025-01-05/file.json:647]: Description can be changed to - "Returned only for provisioned v2 file shares. Returns an approximate used snapshot storage size of the share, in bytes."

File: specification/storage/data-plane/Microsoft.FileStorage/stable/2025-01-05/file.json (hunk 3 above)
Comment (seanmcc-msft) [reply specification/storage/data-plane/Microsoft.FileStorage/stable/2025-01-05/file.json:642]: Fixed.

File: specification/storage/data-plane/Microsoft.FileStorage/stable/2025-01-05/file.json (hunk 4 above)
Comment (seanmcc-msft) [reply specification/storage/data-plane/Microsoft.FileStorage/stable/2025-01-05/file.json:647]: fixed.

File: specification/storage/data-plane/Microsoft.FileStorage/stable/2025-01-05/file.json [hunk 5]
```diff
@@ -559,13 +565,13 @@
               },
               "x-ms-share-next-allowed-provisioned-iops-downgrade-time": {
                 "x-ms-client-name": "NextAllowedProvisionedIopsDowngradeTime",
-                "description": "Return timestamp for provisioned IOPS following existing rules for provisioned storage GiB.",
+                "description": "Returns the current share next allowed provisioned iops downgrade time.",
```
Comment (Mayuri-Gadsing) [inline specification/storage/data-plane/Microsoft.FileStorage/stable/2025-01-05/file.json:568]: Similar change is missing for SetShareProperties API

File: specification/storage/data-plane/Microsoft.FileStorage/stable/2025-01-05/file.json [hunk 6]
```diff
@@ -559,13 +565,13 @@
...
                 "type": "string",
                 "format": "date-time-rfc1123"
               },
               "x-ms-share-next-allowed-provisioned-bandwidth-downgrade-time": {
                 "x-ms-client-name": "NextAllowedProvisionedBandwidthDowngradeTime",
-                "description": "Return timestamp for provisioned throughput following existing rules for provisioned storage GiB.",
+                "description": "Returns the current share next allowed provisioned bandwidth downgrade time.",
```
Comment (Mayuri-Gadsing) [inline specification/storage/data-plane/Microsoft.FileStorage/stable/2025-01-05/file.json:574]: Similar change is missing for SetShareProperties API

File: specification/storage/data-plane/Microsoft.FileStorage/stable/2025-01-05/file.json (hunk 5 above)
Comment (seanmcc-msft) [reply specification/storage/data-plane/Microsoft.FileStorage/stable/2025-01-05/file.json:568]: Fixed.

File: specification/storage/data-plane/Microsoft.FileStorage/stable/2025-01-05/file.json (hunk 6 above)
Comment (seanmcc-msft) [reply specification/storage/data-plane/Microsoft.FileStorage/stable/2025-01-05/file.json:574]: Fixed.

**Review by tg-msft — APPROVED** [review]

**Pull request merged**